pocs/scheduler/observation.py

```diff
@@ -107,6 +109,24 @@ def seq_time(self):
     def seq_time(self, time):
         self._seq_time = time
 
+    @property
+    def directory(self):
+        """Return the directory for this Observation.
+
+        This return the base directory for the Observation. This does *not* include
```

Comment:
This comment makes perfect sense. Given that, and my previous comment, it would seem that Observatory or POCS would be responsible for taking the observation directory and creating the path for a specific camera.

Reply:
In addition to the comments above, the camera could feasibly take an "observation" without the use of POCS or an Observatory. There is a `test_observation` in `test_cameras.py` that does just this. Not sure how useful that is however.